internal/schemautil/userconfig/stateupgrader/v0/kafka/kafka_topic.go

```diff
@@ -209,6 +209,7 @@ var aivenKafkaTopicSchema = map[string]*schema.Schema{
 					Description:      "unclean.leader.election.enable value",
 					Optional:         true,
 					DiffSuppressFunc: schemautil.EmptyObjectDiffSuppressFunc,
+					Deprecated:       "This field is deprecated and won't be available starting from the next major release.",
```

Comment:
I've had an internal talk with @eliax1996, perhaps we would need additional input from them in this GitHub thread on whether this is getting removed or will it still be accepted by the API?

if it's getting removed, we will have to remove it as well, as we have adjusted our development strategy previously to remove fields which no longer work with the API itself

Reply:
We are discussing on the strategy [here](https://aiven.atlassian.net/jira/software/c/projects/HH/boards/61?modal=detail&selectedIssue=HH-1012&assignee=642a0356d774ab7297295922) we want to get rid of this property as soon as possible because it's hard to understand and it's creating a lot of escalations. We will enable our customers only after chatting with us and we will do that only internally by the prune administrative shell.

TLDR: long term customers won't be able to modify this property directly, right now we are planning to remove it from Terraform, the K8s operator, the CLI and the website. We have to discuss how to move on that change also over the `.sh` kafka management scripts and on the MM

Reply:
I have a meeting today just to confirm the strategy, I will come back today on that topic! Let me confirm that we can drop it, in that case we can close the mr and delete directly the field

Reply:
thanks a lot, please let us know if this is getting removed at the end of this talk, and we'll get another PR which removes it instead of deprecating it ready then :)

Reply:
confirmed, this is the plan. Official message [here](https://aiven.atlassian.net/browse/HH-1012?focusedCommentId=159153)